_maps/map_files/RandomZLevels/terrorspiders.dmm

```diff
@@ -15,8 +15,12 @@
 /turf/simulated/wall/rust,
 /area/awaymission/UO71/plaza)
 "ae" = (
-/obj/machinery/atmospherics/pipe/simple/hidden/supply,
 /obj/effect/decal/cleanable/dirt,
+/obj/machinery/atmospherics/pipe/manifold/hidden/supply{
+	dir = 4;
+	initialize_directions = 11;
+	tag = "icon-manifold-b-f (EAST)"
```

Comment:
Little dubious of the tagging of things, pretty sure it gets weird with usage of `\ref` and `UID()`

Is this tagging intentional or just DM being DM?

Reply:
The existing pipes had tags. The edits just re-use existing pipe instances.

Reply:
All the tags have been removed.